### PR TITLE
Create main.js

### PR DESCRIPTION
**Pull request by dmlatesting**
Homework for Day 5

### CODE REVIEW

File: Day-5-Exercises/main.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+console.log(itCompanies.join(', '))
+
+// Check if a certain company exists in the itCompanies array. If it exist return the company else return a company is not found
+let someCompany = 'IBM'
+console.log(`${itCompanies.includes(someCompany)?someCompany:'Company is not found'}`)
+
+// Filter out companies which have more than one 'o' without the filter method
+const tempArr = []
+itCompanies.forEach((element, index) => {    
+    if (element.split('o').length <= 2) {tempArr.push(element)};
```
Comment (jkalandarov) [inline Day-5-Exercises/main.js:81]: You should use the greater than or equal operator to find at least if the length is equal to 2.

File: Day-5-Exercises/main.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+// Filter out companies which have more than one 'o' without the filter method
+const tempArr = []
+itCompanies.forEach((element, index) => {    
+    if (element.split('o').length <= 2) {tempArr.push(element)};   
+})
+//itCompanies = tempArr - If need to save it in initial array
+console.log(tempArr)
+
+// Sort the array using sort() method
+itCompanies.sort
```
Comment (jkalandarov) [inline Day-5-Exercises/main.js]: methods are called by opening parentheses `()` after them.

File: Day-5-Exercises/main.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+const fullStack = frontEnd.concat(backEnd)
+
+console.log(fullStack)
+
+
+//Exercise: Level 3
+
+//The following is an array of 10 students ages:
+const ages = [19, 22, 19, 24, 20, 25, 26, 24, 25, 24]
```
Comment (jkalandarov) [inline Day-5-Exercises/main.js]: try adding 119 to the end of the array, and then sorting. It'll be interesting :)

File: Day-5-Exercises/main.js [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+- Compare the value of (min - average) and (max - average), use abs() method 
+*/
+ages.sort()
+let minAge = ages[0]
+let maxAge = ages[ages.length - 1]
+console.log(ages)
+console.log( `Min age: ${minAge}, Max age: ${maxAge}`)
+
+let sum = 0
+ages.forEach(element => {sum = sum + element})
```
Comment (jkalandarov) [inline Day-5-Exercises/main.js:182]: can you try using `.reduce()` method which does the same job?

**Review by jkalandarov — CHANGES_REQUESTED** [review]

File: Day-5-Exercises/main.js (hunk 1 above)
Comment (dmlatesting) [reply Day-5-Exercises/main.js:81]: As agreed - will leave this solution as correct one

File: Day-5-Exercises/main.js (hunk 2 above)
Comment (dmlatesting) [reply Day-5-Exercises/main.js]: My bet. Fixed

File: Day-5-Exercises/main.js (hunk 3 above)
Comment (dmlatesting) [reply Day-5-Exercises/main.js]: Reworked to sort as integers

File: Day-5-Exercises/main.js (hunk 4 above)
Comment (dmlatesting) [reply Day-5-Exercises/main.js:182]: added alternative option via .reduce()

**Review by jkalandarov — APPROVED** [review]

**Pull request merged**